src/data/STATUSES/root/SHARED.ts

```diff
@@ -25,6 +26,12 @@ export const SHARED = ensureStatuses({
 		icon: 'https://xivapi.com/i/015000/015011.png',
 	},
 
+	TRANSCENDENT: {
+		id: 418,
+		name: 'Transcendent',
+		icon: 'https://xivapi.com/i/015000/015273.png',
+	},
```

Comment:
IIRC this has a max duration of 5s if it matters.

Reply:
I was hesitant to give it a duration given it drops the moment the player uses something. Duno.

Reply:
I kinda agree but there's other statuses people can lose early (ie Meditate, Anatman, any other channeling shit), plus actually removing stuff I guess although that's rare outside tanking dungeons with a derpy healer and poor regen timing.